src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx

```diff
@@ -199,139 +228,117 @@ Global decisions cover a variety of correlation scenarios. Review descriptions t
 
 <figcaption>
   **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Applied intelligence > Incident intelligence > Decisions**: Suggested decision block.
 </figcaption>
 
-If there isn't enough data to see the correlation rate, a link below the percentage estimate will guide you to other sources you can add to get stronger results. If you have less than 5000 incidents per month, you probably won't have suggested decisions.
-
-To enable the suggested decision, click **Activate decision**. If the decision isn't relevant to your needs, click **Dismiss**.
+### Step 2: Correlate context [#correlate-context]
 
-## Create custom decisions [#customize]
+Once you've filtered your data, define the logic used when comparing the incidents' context. You can correlate events based on the following methods:
 
-You can reduce noise and improve correlation by building your own custom decisions. To start building a decision, go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI > Incident intelligence > Decisions**, then click **Add a decision**.
+* Attribute value comparisons with standard operators
+* Attribute value similarity using [similarity algorithms](#algorithms)
+* Attribute value [regex with capture groups](#regex)
+* Entire incident comparisons using similarity or clustering algorithms
 
-When building a decision, steps 1, 2, and 3 are optional on their own, but at least one must be defined in order to create a decision.
+<img
+  title="Decision - correlate context"
+  alt="A screenshot of the decision builder showing how to apply contextual correlation."
+  src={decisionBuilderContextual}
+/>
 
-<table id="decision-guidelines">
-  <thead>
-    <tr>
-      <th style={{ width: "250px" }}>
-        Option
-      </th>
+<figcaption>
+  Click **Preview** to view [simulated](#simulations) correlation efficiency (from recent incidents) for the current decision logic.
+</figcaption>
 
-      <th>
-        Description
-      </th>
-    </tr>
-  </thead>
+### Step 3: Apply topology correlation [#topology-correlation]
 
-  <tbody>
-    <tr>
-      <td>
-        Step 1: Filter your data (optional)
-      </td>
+For automatic topology correlation, make sure your telemetry data is collected by [New Relic agents](/docs/new-relic-solutions/new-relic-one/install-configure/compatibility-requirements-new-relic-agents-products/). Learn more about [topology correlation out-of-the-box](#topology-requirements).
 
-      <td>
-        In this step you'll define your filters. Remember correlation occurs between two incidents. If no filters are defined, all incoming incidents will be considered by the decision.
+In addition, you can setup topology data via our NerdGraph `aiTopologyCollector`. (Search for `aiTopology` in the [NerdGraph GraphiQL explorer](https://api.newrelic.com/graphiql)). This allows any topology-related decision to be matched with your topology data. Learn more about [setting up topology correlation](#topology).
```

Comment:
we have a doc for this https://docs.newrelic.com/docs/apis/nerdgraph/examples/topology-nerdgraph-tutorial/ that we should reference here instead of pointing them to the nerdgraph explorer.

Reply:
thank you!